src/post/content.ts

```diff
@@ -170,6 +198,12 @@ export class ContentPreparer {
         return `${text.substring(0, charLimit - 3)}...`;
     }
 
+    private parseMentions(content: string): Set<string> {
+        const handleRegex = /@[\w.-]+@[\w-]+\.[\w.-]+/g;
+        const mentions = content.match(handleRegex) || [];
+        return new Set(mentions.filter(isHandle));
+    }
```

Comment:
We're checking against two regexes here (handleRegex, then the one defined in isHandle helper) - feels redundant?

Should we extract the regex in `isHandle` as a constant e.g. `export const HANDLE_REGEX = /^@([\w.-]+)@([\w-]+\.[\w.-]+[^.])$/` and then use it in `isHandle` and in `parseMentions`?

Reply:
Done https://github.com/TryGhost/ActivityPub/pull/666/commits/ac30087a15cead75ae3a68a74bc6ae8eb5857d03